OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/DiscussionCommentsAdapter.java

```diff
@@ -121,24 +130,33 @@ public void onBindViewHolder(RecyclerView.ViewHolder viewHolder, int position) {
             iconDrawable = new IconDrawable(context, FontAwesomeIcons.fa_comment)
                     .sizeRes(context, R.dimen.edx_small)
                     .colorRes(context, R.color.edx_brand_gray_base);
+            holder.discussionCommentCountReportTextView.setVisibility(View.VISIBLE);
```

Comment:
Hmm, so thinking upon this, we aren't even recycling the views between each other, nor can they be recycled, since they have different layout, even if the views match for the most part. So it seems that you can rely on this always being visible by default, and don't need to set it's visibility explicitly.